ParseLiveQuery/build.gradle

```diff
@@ -24,18 +24,18 @@ android {
 }
 
 dependencies {
-    api "com.github.parse-community.Parse-SDK-Android:parse:1.19.0"
-    api 'com.squareup.okhttp3:okhttp:3.12.1'
+    api "com.github.parse-community.Parse-SDK-Android:parse:1.22.1"
+    api "com.squareup.okhttp3:okhttp:3.14.4"
```

Comment:
Is there something about OkHttp 3.14.4 that was necessary for LiveQuery to be updated?

Note that Parse-SDK-Android explicitly adds a note that 3.13 and newer should be avoided, because OkHttp 3.13 bumps the minSdKVersion to Android 5.0:  https://github.com/parse-community/Parse-SDK-Android/blob/master/parse/build.gradle#L31-L32

If 3.14 isn't an absolute requirement for something, I think LiveQuery should follow the same reasoning here, and stick to 3.12. Otherwise line 8 minSdkVersion 15 is wrong, or at the very least it's misleading.

I opened #113 to track this.